pkg/apis/workspaces/v1alpha1/component_plugin_conversion.go

```diff
@@ -27,6 +27,7 @@ func convertPluginComponentTo_v1alpha2(srcComponent *Component, destComponent *v
 	destComponent.Name = pluginKey
 
 	for _, srcCommand := range src.Commands {
+		srcCommand := srcCommand
```

Comment:
This line appears to be a no-op?

Reply:
it ensures the address is not reused.  It's one of the approaches you can use to fix this issue (the other is to use an index) according to this discussion: https://stackoverflow.com/questions/62446118/implicit-memory-aliasing-in-for-loop.

Reply:
Ah makes sense, I didn't realize we used `&srcCommand` below.